qa/tasks/cram.py

```diff
@@ -134,8 +134,10 @@ def _run_tests(ctx, role):
     """
     assert isinstance(role, str)
     PREFIX = 'client.'
-    assert role.startswith(PREFIX)
-    id_ = role[len(PREFIX):]
+    if role.startswith(PREFIX):
```

Comment:
please replace "cram" with "qa/tasks" in the title of the commit.

Reply:
Sure, thanks.